### DISCUSSION TITLE
Fetch many ids at once

### DISCUSSION THREAD

**Discussion by ghost** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#25] [Ideas]
**Is your feature request related to a problem? Please describe.**
This is related to #11 due to Instagram rate limit. 

**Describe the solution you'd like**
Instead of sending many request one by one into the api, it would be better to just send one request that will achieve the same thing. 

For example I subscribed into 16278726 and 217723373, bbcnews and cnn respectively.
So instead of sending GET requests for each `https://i.instagram.com/api/v1/feed/reels_media/?reel_ids=16278726` and `https://i.instagram.com/api/v1/feed/reels_media/?reel_ids=217723373`, you can just send 1 GET request of `https://i.instagram.com/api/v1/feed/reels_media/?reel_ids=16278726&reel_ids=217723373`.

**Additional context**
The `reels_media` array returned from this solution will be a list of each subscriptions Instagram Stories, so if bbcnews and cnn both has an active Instagram story `reels_media[0]` will be bbcnews and `reels_media[1]` will be cnn. Keep in mind that Instagram account that has no active Instagram story won't be listed in `reels_media`.

I'd like to open a PR, but I'm not really confident with my understanding of javascript.

**Comment by chriskyfung** [comment]
Thanks for your suggestion!

I believe the current version should be ample for regular usages, such as news cutting and following close friends' stories.
It won't reach the rate limit unless requesting the API with a high frequency, e.g. run the script every minute for web scraping sites and invading people's privacy.
